backend/src/StdLibExecution/Libs/Math.fs

```diff
@@ -18,15 +18,7 @@ let fn = fn modules
 let constant = constant modules
 
 let constants : List<BuiltInConstant> =
-  [ { name = constant "pi" 0
-      typ = TFloat
-      description =
-        "Returns an approximation for the mathematical constant {{π}},
-        the ratio of a circle's circumference to its diameter."
-      body = DFloat System.Math.PI
-      deprecated = NotDeprecated }
-
-    { name = constant "tau" 0
+  [ { name = constant "tau" 0
```

Comment:
I think you meant to remove tau from here?

Reply:
Yes, I wasn't sure if using a direct assignment of the tau value instead of `2*pi` was acceptable. I'll remove it in a subsequent PR